Rubberduck.Core/UI/Command/InspectionResultsCommand.cs

```diff
@@ -9,9 +10,9 @@ namespace Rubberduck.UI.Command
     [ComVisible(false)]
     public class InspectionResultsCommand : CommandBase
     {
-        private readonly IDockablePresenter _presenter;
+        private readonly InspectionResultsDockablePresenter _presenter;
```

Comment:
Why have the concrete type here?

Reply:
This was shortly discussed in chat yesterday ([permalink](https://chat.stackexchange.com/transcript/message/46785598#46785598)) - bottom line, there's definitely a design problem with these, but in the current state of things we don't have tests at the command level (right?), so the decoupling is kind of artificial and the interfaces are kind of violating LSP anyway - injecting `BarDockablePresenter` in `FooCommand` doesn't make any sense, the coupling is fine for now I think, until we come up with a better design.